tests/selenium/README.md

```diff
@@ -0,0 +1,69 @@
+# Tests for Mozilla's Treeherder
```

Comment:
Let's reword the README to the context of a director rather than a repository. Also, we can remove all the badges.